codec_test.go

```diff
@@ -403,28 +400,109 @@ func BenchmarkParseV4(b *testing.B) {
 	}
 }
 
-var seedFuzzCorpus = flag.Bool("seed_fuzz_corpus", false, "seed fuzz test corpus")
+const uuidPattern = "[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"
```

Comment:
Should this also include capital `A-F` in the match patterns since those would still be valid UUID strings?

Reply:
Thanks, nice catch